deploy-board/deploy_board/webapp/deploy_views.py

```diff
@@ -28,6 +28,36 @@
 DEFAULT_PAGE_SIZE = 30
 DEFAULT_ONGOING_DEPLOY_SIZE = 10
 
+def _get_running_deploys_count(request):
+    deploy_states = ["RUNNING"]  # current running deploys
+    page_size = 1                # only need to return 1 detail, the return structure has the "total" value
+    deployResult = deploys_helper.get_all(request, deployState=deploy_states, pageSize=page_size)
+    return deployResult['total']
+
+def _get_sidecars(request):
```

Comment:
just a check here, "sidecar" is the term we want to use here?

Reply:
yes. I think it is better to have this concept as we've been using it everywhere